azure-openai-model-provider/tool.gpt

```diff
@@ -9,4 +9,12 @@ Credential: ../model-provider-credential as azure-openai-model-provider
 
 ---
 !metadata:*:icon
-/admin/assets/azure_openai_icon.svg
\ No newline at end of file
+/admin/assets/azure_openai_icon.svg
+
+---
+Name: validate
+Description: Validate Azure OpenAI credentials
+Metadata: envVars: OBOT_AZURE_OPENAI_MODEL_PROVIDER_ENDPOINT,OBOT_AZURE_OPENAI_MODEL_PROVIDER_CLIENT_ID,OBOT_AZURE_OPENAI_MODEL_PROVIDER_CLIENT_SECRET,OBOT_AZURE_OPENAI_MODEL_PROVIDER_TENANT_ID,OBOT_AZURE_OPENAI_MODEL_PROVIDER_SUBSCRIPTION_ID,OBOT_AZURE_OPENAI_MODEL_PROVIDER_RESOURCE_GROUP
+Metadata: optionalEnvVars: OBOT_AZURE_OPENAI_MODEL_PROVIDER_API_VERSION=2024-10-21
```

Comment:
nit: we don't need this here. The metadata fields on the model provider tool above are all that is necessary.